	<!DOCTYPE html>
<html>

<head>
	
<script>
var host = "chavezlab.org";
if ((host == window.location.host) && (window.location.protocol != "https:"))
    window.location.protocol = "https";
</script>
	
<meta charset="utf-8">
<meta name="viewport" content="width=device-width, initial-scale=1">
<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
<script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.0/js/bootstrap.min.js"></script>
<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.5.0/css/all.css" integrity="sha384-B4dIYHKNBt8Bc12p+WXckhzcICo0wtJAoU8YZTY5qE0Id1GSseTk6S+L3BlXeVIU" crossorigin="anonymous">

<!-- Loading third party fonts -->
<link href="https://fonts.googleapis.com/css?family=Roboto:300,400,700|" rel="stylesheet" type="text/css">
<link href="/fonts/font-awesome.min.css" rel="stylesheet" type="text/css">

<!-- Loading main css file -->
<link rel="stylesheet" href="/css/styles.css">
</head>
<body>


<!-- 1. Write navigation bar with "nav" element-->
<nav class="navbar navbar-default banner">
	<div class="container-fluid">
	  <!-- Brand and toggle get grouped for better mobile display -->
	  <div class="navbar-header">
		<button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1" aria-expanded="false">
		  <span class="sr-only">Toggle navigation</span>
		  <span class="icon-bar"></span>
		  <span class="icon-bar"></span>
		  <span class="icon-bar"></span>
		</button>
		<a class="navbar-brand" href="/">Chavez Lab</a>
	  </div>
  
	  <!-- Collect the nav links, forms, and other content for toggling -->
	  <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
		<ul class="nav navbar-nav navbar-right navbar-nav-primary">
			<li><a href="/">HOME</a></li>
			<li><a href="/research">RESEARCH</a></li>
			<li><a href="/people">PEOPLE</a></li>
			<li><a href="/software">SOFTWARE & DATA</a></li>
			<li><a href="/publications">PUBLICATIONS</a></li>
			<li><a href="/misc">MISCELLANEOUS</a></li>
		</ul>
	  </div><!-- /.navbar-collapse -->
	</div><!-- /.container-fluid -->
  </nav>

<!-- 4. Add main title and subtitle-->

<header class="bg-primary" style="padding: 50px 0 50px">
	<div class="container">
		<h1 style="font-size: 60px !important; font-family: 'Montserrat', sans-serif; letter-spacing: 2px; font-weight: 100;">Our Major Goals</h1>
	</div>
</header>

<div class="container" style="padding-top: 50px;">
	<p class="text-justify" style="font-size: 16px;">
		<p>
			The primary goal of this proposal is to investigate the role of circular extrachromosomal DNA (ecDNA) as a potential driver of intratumoral heterogeneity and treatment resistance in medulloblastoma, the most prevalent pediatric malignant brain tumor. Intratumoral heterogeneity stands as a leading determinant of therapeutic resistance and treatment failure, contributing significantly to poor overall survival in cancer patients. Despite its critical implications, the functional relevance of ecDNA in driving tumor heterogeneity and treatment resistance in medulloblastoma remains largely unexplored.
		  </p>
		  <p>
			To analyze the clinical impact of ecDNA across different molecular subgroups of medulloblastoma, we have assembled a multi-institutional cohort comprising Whole Genome Sequencing data from 468 medulloblastoma patient samples. Using innovative computational methods for the detection and reconstruction of ecDNA, our findings indicate the presence of ecDNA in 18% of patients, with a significant association with poorer outcomes. Furthermore, our observations reveal that individual medulloblastoma tumors often harbor multiple variants of ecDNA, each containing different amplified oncogenes along with co-amplified non-coding regulatory DNA ('enhancers').
		  </p>
		  <p>
			The central hypothesis of this research posits that ecDNA plays a pivotal role in driving intratumoral heterogeneity and treatment resistance, particularly in high-risk medulloblastoma patients. This overarching hypothesis will be rigorously tested through three specific aims: investigating the molecular evolution of ecDNA as a potential driver of treatment resistance, evaluating combinatorial therapies targeted against mechanisms of ecDNA formation and clustering to reduce treatment resistance, and probing medulloblastoma tumor-dependencies by functionally inhibiting coding and non-coding regulatory DNA co-amplified on ecDNA.
		  </p>
		  <p>
			The proposed research brings forth technical, conceptual, and biological innovations, employing advanced methods such as the analysis of ecDNA at the single-cell level using novel imaging and multiome single-nucleus sequencing techniques in both medulloblastoma tumors and patient-derived xenograft (PDX) models. With the current standard treatment for children with medulloblastoma causing developmental disorders, neurological damage, and secondary metastases, there is an urgent need for novel therapeutic approaches. Our approach aims to assess the impact of standard-of-care treatments on the molecular evolution of ecDNA and to functionally test novel combination treatments targeted against ecDNA genesis and clustering. These preclinical studies have the potential to unveil novel mechanisms by which ecDNA contributes to the pathogenesis of medulloblastoma, identify new scientific leads for improved treatments, and ultimately improve patient outcomes.
		  </p>
		
		
		
		 
	 </br></br>
	 Recently, we have demonstrated how to leverage epigenetic information such as DNA methylation and enhancer profiling in pediatric brain tumors and normal human tissues to identify clinically relevant tumor subgroups, oncogenic enhancers, transcription factors, and pathways amenable to pharmacologic targeting. To reveal regulatory circuitries disturbed in childhood brain tumors, the lab generates and integrate public high-dimensional data from primary tumors and patient-derived models. By integrating these heterogeneous data types, we aim to identify oncogenic mechanisms and novel tumor-dependency genes and pathways that may provide new targets for improved therapeutic approaches. In addition, the lab develops and applies new statistical and machine learning tools for the quantitative and integrative analysis of heterogeneous genetic and epigenetic data.
	</br></br>
		
	
		
	</p>
	
</div>


<footer class="site-footer">
	<div class="container">
	  <div class="row">
		<div class="col-sm-12 col-md-6">
		  <!-- <h6>About</h6> -->
		  <p class="text-justify">
		  </p>
		  <img src="/images/Sanford_Burnham_Prebys_Medical_Discovery_Institute_Logo.png" alt="" width ='195px' height='50px' hspace="20">
		  <img src="/images/UCSanDiegoLogo-White.png" alt="" width ='100px' height='50px' hspace="20">
		  <img src="/images/Rady_Children's_Hospital_logo.svg" alt="" width ='100px' height='120px' hspace="20">
		</div>
		<div class='col-md-3'></div>

		<div class="col-xs-6 col-md-3">
		  <h6>Quick Links</h6>
		  <ul class="footer-links">
			<li><a href="/">Home</a></li>
        	<li><a href="/research">Research</a></li>
			<li><a href="/people">People</a></li>
			<li><a href="/software">Software & Data</a></li>
			<li><a href="/publications">Publications</a></li>
			<li><a href="/misc">Miscellaneous</a></li>
		  </ul>
		</div>
	  </div>
	  <hr>
	</div>
</footer>

</body>
</html>